CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
I'm not sure this is true given that `0.13` is not a valid Semantic version 😆 .

Though I'd be in favour of moving to semantic versions.

Reply:
Removed the line and we will fix the versioning on the next release